lib/buffer.js

```diff
@@ -292,7 +292,7 @@ Buffer.concat = function(list, length) {
 
   var buffer = Buffer.allocUnsafe(length);
   var pos = 0;
-  for (let i = 0; i < list.length; i++) {
+  for (i = 0; i < list.length; i++) {
```

Comment:
Just wondering why no var here.  Looks like it was there previously https://github.com/nodejs/node/pull/5819/files